climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -92,7 +92,11 @@ class TCForecast(TCTracks):
     data : list of xarray.Dataset
         Same as in parent class, adding the following attributes
         - ensemble_member (int)
-        - is_ensemble (bool; if False, the simulation is a high resolution deterministic run
+        - is_ensemble (bool; if False, the simulation is a high resolution
+                       deterministic run)
+        - forecast_time (numpy.datetime64): timepoint of the initialisation of
```

Comment:
When you have an ensemble, must all forecast_time be identical?

Reply:
good question. the answer is probably yes. i'm not sure when you'd want to mix different forecast initialisation times in a single Hazard object.